dev/src/sidebar-config.js

```diff
@@ -36,6 +36,7 @@ export const sidebarConfig = [
     title: 'Layout',
```

Comment:
I have just found a component in this package that looks very much like our Badge. It's called `Tag` and is listed under "Communication" section. 
1. Would it make sense to put Badge under "Communication", too?
2. Should we maybe add some deprecation message to use `Badge` instead of `Tag`?

<img width="794" alt="Screenshot 2023-09-20 at 12 30 13" src="https://github.com/warp-ds/vue/assets/41303231/ee98dfb9-2dfd-4662-9fa6-696649b41df2">

<img width="794" alt="Screenshot 2023-09-20 at 12 31 28" src="https://github.com/warp-ds/vue/assets/41303231/e0bec02c-c581-4c2f-9bd9-afa5160f4b16">

Reply:
1. Yes, that makes sense. 👍 

2. I have no idea... this is what Figma says:
> Badges and tags are visually similar, but tags are interactive and badges are not...

It is also called "Ribbon" and there are some implementations out there that uses those component-classes for custom made "badges"... Makes me want to just drop everything when it's not the first time that you do something and try to understand the mess and asks for input... then finish it, the way you understand it... only to be told afterwards that it is "wrong". 😞

Reply:
I'm sorry to not have spotted it sooner :( It has also been confusing to me last time I looked at it - one and the same component being called ribbon, badge and tag. I think it all came from the fact that different people were developing Vue and React component libraries in Fabric. I tried aligning it with @adidick a few months back but we didn't land on anything, just got more confused and blocked with different approaches to the naming and so on 😅 I thought this was discussed again between you two and that you landed on the "Badge" as the way to go. If that's the case then that's great because we need that component everywhere, with just 1 name 😄 The only thing that's left is removing the deprecated Tag in Vue, but we could do that some time in the future. I only thought we could make a note/suggestion that Tag is deprecated in favour of Badge.

> Badges and tags are visually similar, but tags are interactive and badges are not...
Where did you find this description? I can't see that the two are different in this way, at least based on the implementation. But maybe I'm reading vue code wrong 👀

Reply:
![image](https://github.com/warp-ds/vue/assets/7531505/d34f966b-b0e9-47e1-8ca0-1f1b53666e8a)

Reply:
But no, you are not reading the vue code wrong... it seems to be doing the same thing as badge.

Reply:
Sounds to me like good news 🙂 We wouldn't be able to rename Tag anyway due to backward compatibility (who knows who uses that except us in the vue docs). I think we're free to only display Badge in the list of components both in the tech docs and in these vue docs if that helps.

Reply:
Anna did a code search for the Tag component and couldn't find anything. The component is broken and would, as an example, not get base styling if not adding a `primary` prop to it. Which doesn't make sense. I removed it from the sidebar in the internal docs in this repo.

Reply:
Wonderful! 🙌